paderbox/array/interval/core.py

```diff
@@ -358,6 +358,32 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
+    def as_tuple(self):
+        """
```

Comment:
Can you add a short description of what this function is useful for?

Reply:
Sure, I thought the name was verbose enough to convey what it's meant for, but I think a description won't hurt.